src/wallet/rpcwallet.cpp

```diff
@@ -758,17 +758,17 @@ static UniValue getbalance(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_METHOD_DEPRECATED, "dummy first argument must be excluded or set to \"*\".");
     }
 
-    int min_depth = 0;
     if (!request.params[1].isNull()) {
-        min_depth = request.params[1].get_int();
+        int min_depth = request.params[1].get_int();
```

Comment:
nit: seems strange to define a variable in this scope which is just going to get used once. How about:

```
if (!request.params[1].isNull() && request.params[1].get_int() != 0) {
    throw ...
```

Reply:
Sure.